js/modules/loki_app_dot_net_api.js

```diff
@@ -1377,6 +1378,7 @@ class LokiPublicChannelAPI {
         message: messageData,
       });
     });
+    /* eslint-enable no-param-reassign */
```

Comment:
that one is useless

Reply:
doesn't it re-enable the checking, that we want on because we have that rule enabled?

Reply:
Oh yes you're right, I thought the `eslint-disable no-param-reassign` was only set for next line so you wouldn't need to re-enable it, but it's netter this way because you have several lines on you need to disable this rule.
Good to me